cli/commands/login/login.go

```diff
@@ -37,6 +37,7 @@ import (
 
 const (
 	// URLFlag is the viper flag for the server url
+	// TODO(oxisto): Unused constant?
 	URLFlag = "url"
```

Comment:
Do we need that constant @oxisto ? It seems to be unused.

Reply:
Keep it for now. We need to make all of those URLs configurable at some point.